.github/workflows/merge.yaml

```diff
@@ -0,0 +1,185 @@
+name: OpenFisca-Core / Deploy package to PyPi & Conda
+
+on:
+  push:
+    branches: [master]
+
+concurrency:
+  group: ${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  setup:
+    strategy:
+      fail-fast: true
+      matrix:
+        os: [ubuntu-20.04, windows-latest]
```

Comment:
Can you add a comment to explain why you use old ubuntu-20.04 instead of ubuntu-22.04 ?
Because the Python version between Windows-latest and Ubuntu 22.04 did not match, cf https://raw.githubusercontent.com/actions/python-versions/main/versions-manifest.json ?

Reply:
I think in the time between this PR was open Github also updated windows.